k9mail/src/main/res/xml/global_preferences.xml

```diff
@@ -415,12 +415,12 @@
 
         <org.openintents.openpgp.util.OpenPgpAppPreference
             android:persistent="false"
-            android:key="crypto_app"
+            android:key="openpgp_provider"
```

Comment:
Hmmm okay. Isn't this then true for all keys inside the xml files? At least persistent is always "false". Maybe the naming should be unified in another PR if this is the case. Currently it is confusing.